tsconfig.json

```diff
@@ -1,16 +1,20 @@
 {
   "extends": "./.svelte-kit/tsconfig.json",
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
Why do we want this? Typically this is solved by doing `* as object` because the `export` is not an actual default module.

Is this because a new library is using a commonjs export and not a `export default`? I'm guessing this was a debug of why `xss` wasn't importing correctly?

Reply:
This was to allow the import of JSON files, as they don't have a default export. Do you know of another way to import JSON files without using this config setting?

Reply:
I think all that would be needed is  `resolveJsonModule: true`

https://www.typescriptlang.org/tsconfig#resolveJsonModule

Reply:
Hmm, okay, I'll have to try that.